js/remove.ts

```diff
@@ -3,44 +3,24 @@ import * as msg from "gen/msg_generated";
 import * as flatbuffers from "./flatbuffers";
 import * as dispatch from "./dispatch";
 
-/** Removes the named file or (empty) directory synchronously. Would throw
+/** Removes the named file or directory synchronously. Would throw
  * error if permission denied, not found, or directory not empty.
  *
  *       import { removeSync } from "deno";
- *       removeSync("/path/to/empty_dir/or/file");
+ *       removeSync("/path/to/empty_dir/or/file", true);
  */
-export function removeSync(path: string): void {
-  dispatch.sendSync(...req(path, false));
+export function removeSync(path: string, recursive = false): void {
+  dispatch.sendSync(...req(path, recursive));
 }
 
-/** Removes the named file or (empty) directory. Would throw error if
+/** Removes the named file or directory. Would throw error if
  * permission denied, not found, or directory not empty.
  *
  *       import { remove } from "deno";
- *       await remove("/path/to/empty_dir/or/file");
+ *       await remove("/path/to/empty_dir/or/file", true);
  */
-export async function remove(path: string): Promise<void> {
-  await dispatch.sendAsync(...req(path, false));
-}
-
-/** Recursively removes the named file or directory synchronously.  Would throw
- * error if permission denied or not found.
- *
- *       import { removeAllSync } from "deno";
- *       removeAllSync("/path/to/dir/or/file");
- */
-export function removeAllSync(path: string): void {
-  dispatch.sendSync(...req(path, true));
-}
-
-/** Recursively removes the named file or directory. Would throw error if
- * permission denied or not found.
- *
- *       import { removeAll } from "deno";
- *       await removeAll("/path/to/dir/or/file");
- */
-export async function removeAll(path: string): Promise<void> {
-  await dispatch.sendAsync(...req(path, true));
+export async function remove(path: string, recursive = false): Promise<void> {
```

Comment:
Should we pass an options object `{ recursive: true }` instead of `true`?
This makes it easier to extend the interface
At the same time, `true` will make people confused

Reply:
@axetroy Got it.